examples/App.js

```diff
@@ -52,7 +52,7 @@ const EXAMPLES = {
   dndOutsideSource: 'Addon: Drag and drop (from outside calendar)',
 }
 
-const DEFAULT_EXAMPLE = 'basic'
+const DEFAULT_EXAMPLE = 'rendering'
```

Comment:
What is this variable used for?

Reply:
This is used for the first example that shows up first when you starts it. Because I've not changed any other examples, this should show up first to demonstrates the library's capabilities after customized